boojum-cuda/Cargo.toml

```diff
@@ -3,14 +3,15 @@ name = "boojum-cuda"
 version = "0.1.0"
 edition = "2021"
 build = "build/main.rs"
+license = "MIT OR Apache-2.0"
 
 [build-dependencies]
-boojum = { git = "https://github.com/matter-labs/boojum.git", branch = "main" }
+boojum = { git = "https://github.com/matter-labs/era-boojum.git", branch = "main" }
 cmake = "^0"
 itertools = "^0"
 
 [dependencies]
-boojum = { git = "https://github.com/matter-labs/boojum.git", branch = "main" }
+boojum = { git = "https://github.com/matter-labs/era-boojum.git", branch = "main" }
```

Comment:
We shouldn't merge with this until we're good to do all of [this](https://matter-labs-workspace.slack.com/archives/C05ATRT8LRE/p1695363194910559). Let's leave this part in it's own PR.

Reply:
I think era-boojum-cuda isn't used by anyone yet, so it's fine to merge.
Also it can't be used with private repo as dependency.

Reply:
@yorik I think in general we should be careful flipping before we make the decision to flip. In any case it's merged now so we can flip.